README.md

```diff
@@ -214,6 +217,27 @@ And then installing again:
 ./install --force
 ```
 
+#### Missing symbols & ligatures in Vim (Linux)
+
+If symbols appear corrupted in Vim, first ensure that your terminal emulator supports UTF-8. Then, check to see if you've properly configured your system-wide UTF-8 support.
+
+Issue this test command:
+
+```bash
+export LC_ALL=en_US.UTF-8
+```
+
+Then run `vim`. Powerline symbols & statusline ligatures should be working.
```

Comment:
I was referring to the cursive "l" and "c" line / column indicators appearing on the bottom right of the statusline - without UTF-8 support, they didn't appear as expected.